openshift.spec

```diff
@@ -275,7 +414,168 @@ fi
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.kubelet-client.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.server.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.server.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-master.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-master.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-master.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-registry.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-registry.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-registry.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-router.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-router.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-router.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/policy.json
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/serviceaccounts.private.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/serviceaccounts.public.key
+
+%post -n atomic-enterprise-master
+%systemd_post %{basename:atomic-enterprise-master.service}
+# Create all-in-one master configs
+%{_bindir}/atomic-enterprise start master --write-config=/etc/origin/allinone/master
+# Create all-in-one node configs
+%{_bindir}/oadm create-node-config --node-dir=/etc/origin/allinone/node/ --node=localhost --hostnames=localhost,127.0.0.1 --node-client-certificate-authority=/etc/origin/allinone/master/ca.crt --signer-cert=/etc/origin/allinone/master/ca.crt --signer-key=/etc/origin/allinone/master/ca.key --signer-serial=/etc/origin/allinone/master/ca.serial.txt --certificate-authority=/etc/origin/allinone/master/ca.crt
+
+%preun -n atomic-enterprise-master
+%systemd_preun %{basename:atomic-enterprise-master.service}
+%systemd_preun %{basename:atomic-enterprise-allinone.service}
+
+%postun -n atomic-enterprise-master
+%systemd_postun
+
+%files -n atomic-enterprise-node
+%defattr(-,root,root,-)
+%{_unitdir}/atomic-enterprise-node.service
+%config(noreplace) %{_sysconfdir}/sysconfig/atomic-enterprise-node
+%config(noreplace) /etc/origin/node
+
+%post -n atomic-enterprise-node
+%systemd_post %{basename:atomic-enterprise-node.service}
+
+%preun -n atomic-enterprise-node
+%systemd_preun %{basename:atomic-enterprise-node.service}
+
+%postun -n atomic-enterprise-node
+%systemd_postun
+
+
+%files -n atomic-enterprise-sdn-ovs
+%defattr(-,root,root,-)
+%{_bindir}/atomic-enterprise-sdn-kube-subnet-setup.sh
+%{kube_plugin_path}/atomic-enterprise-ovs-subnet
+%{_prefix}/lib/systemd/system/atomic-enterprise-node.service.d/atomic-enterprise-sdn-ovs.conf
+%{_prefix}/lib/systemd/system/docker.service.d/docker-sdn-ovs.conf
```

Comment:
I believe `%{_prefix}/lib/systemd/system/` should be `%{_unitdir}/` (all the occurrences).